xbmc/video/VideoDatabase.cpp

```diff
@@ -4951,47 +4953,34 @@ bool CVideoDatabase::GetArtForAsset(int assetId,
 
     std::string sql;
 
-    if (mediaType == MediaTypeVideoVersion)
+    if (mediaType == MediaTypeMovie)
     {
-      // MAYBE: use more compact and readable non-ANSI SQL extension where (,) in ()?
-      sql = PrepareSQL("SELECT art.media_type, art.type, art.url "
-                       "FROM art "
-                       "LEFT JOIN videoversion vv "
-                       "  ON art.media_id = vv.idMedia AND art.media_type = vv.media_type "
-                       "WHERE (art.media_id = %i AND art.media_type = '%s') "
-                       "OR(vv.idFile = %i) ",
-                       assetId, MediaTypeVideoVersion, assetId);
+      sql = PrepareSQL(
+          "SELECT type,url "
+          "FROM art "
+          "  JOIN movie "
+          "  ON art.media_id = movie.idFile AND art.media_type = '" MediaTypeVideoVersion "' "
+          "WHERE idMovie = %i ",
+          mediaId);
     }
     else
     {
-      sql = PrepareSQL("SELECT media_type, type, url "
-                       "FROM art "
-                       "WHERE (media_id = %i AND media_type = '%s') "
-                       "OR (media_id = %i AND media_type = '%s')",
-                       assetId, MediaTypeVideoVersion, ownerMediaId, mediaType.c_str());
+      sql = PrepareSQL("SELECT type, url FROM art WHERE media_id=%i AND media_type='%s'", mediaId,
+                       mediaType.c_str());
     }
 
     m_pDS2->query(sql);
     while (!m_pDS2->eof())
     {
-      if (m_pDS2->fv(0).get_asString() == MediaTypeVideoVersion)
-      {
-        // version data has priority over owner's data, used as fallback.
-        art[m_pDS2->fv(1).get_asString()] = m_pDS2->fv(2).get_asString();
-      }
-      else
-      {
-        // insert if not yet present
-        art.insert(make_pair(m_pDS2->fv(1).get_asString(), m_pDS2->fv(2).get_asString()));
-      }
+      art.insert(make_pair(m_pDS2->fv(0).get_asString(), m_pDS2->fv(1).get_asString()));
```

Comment:
While you are at it, could you please add the `std` namespace to the `make_pair`call?

Reply:
sure.